packages/clay-css/gulpfile.js

```diff
@@ -14,7 +14,7 @@ var config = {
 	SRC_GLOB: 'src/**/*'
 };
 
-var bootstrapPath = path.dirname(require.resolve('bootstrap/package.json'));
+var bootstrapPath = path.join('.', 'src', 'bootstrap.scss');
```

Comment:
I imagine this might break the build https://github.com/liferay/clay/blob/6f4ba2f956ada96130fedaded52b5ec77f2606a2/packages/clay-css/gulpfile.js#L67-L80, can you check if it's breaking the build?

Reply:
You're right let me update all the paths.